azurerm/internal/services/kusto/kusto_eventhub_data_connection_resource.go

```diff
@@ -55,6 +55,16 @@ func resourceArmKustoEventHubDataConnection() *schema.Resource {
 				ValidateFunc: validateAzureRMKustoClusterName,
 			},
 
+			"compression": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
```

Comment:
is there a default value for this? I presume `None`?

Reply:
Seems like it. Set default to None and updated docs.